doc/handle-ASIC-SDK-health-event/handle-ASIC-SDK-health-event.md

```diff
@@ -0,0 +1,635 @@
+
+If the category-list is `none`, none category is suppressed and all the categories will be notified for `severity`.
+
+If the category-list is `all`, all the categories are suppressed and none category will be notified for `severity`.
+
+The ASIC/SDK health events whose `category` is in `category-list` with the `severity` will not be reported by the vendor SAI once the corresponding SAI attributes are set.
+But the events that were reported after the command is executed but before the SAI attributes are set will be handled by orchagent and pushed into `STATE_DB.ASIC_SDK_HEALTH_EVENT_TABLE` as usual.
+
+Eg. the following command will suppress `notice` events with category `asic_hw` and `cpu_hw`:
+
+`config asic-sdk-health-event suppress notice asic_hw,cpu_hw`
+
+The following error message will be shown if a customer configures it on a platform that does not support it.
+
+`ASIC/SDK health event is not supported on the platform`
+
+The following error message will be shown if a customer suppresses a severity which is not supported on the platform.
+
+`Suppressing ASIC/SDK health {severity} event is not supported on the platform`
+
+##### Display the ASIC/SDK health events
+
+Command `show asic-sdk-health-events received` is introduced to display the ASIC/SDK health events as a table.
+An example of the output is as below:
+
+```
+admin@sonic:~$ show asic-sdk-health-events
+Time                 Severity     Category   Description
+-------------------  -----------  ---------  -----------------
+2023-10-20 05:07:34  fatal        firmware   Command timeout
+2023-10-20 03:06:25  fatal        software   SDK daemon keep alive failed
+2023-10-20 05:07:34  fatal        asic_hw    Uncorrectable ECC error
+2023-10-20 01:58:43  notice       asic_hw    Correctable ECC error
+```
+
+An example of the output on a multi ASIC system:
+
+```
+admin@sonic:~$ show asic-sdk-health-events
+asic0:
+Time                 Severity     Category   Description
+-------------------  -----------  ---------  -----------------
+2023-10-20 05:07:34  fatal        firmware   Command timeout
+2023-10-20 03:06:25  fatal        software   SDK daemon keep alive failed
+asic1:
+Time                 Severity     Category   Description
+-------------------  -----------  ---------  -----------------
+2023-10-20 05:07:34  fatal        asic_hw    Uncorrectable ECC error
+2023-10-20 01:58:43  notice       asic_hw    Correctable ECC error
+```
+
+The following error message will be shown if a customer executes the command on a platform that does not support it.
+
+`ASIC/SDK health event is not supported on the platform`
+
+##### Display the ASIC/SDK health suppress configuration
+
+Command `show asic-sdk-health-events suppressed-category-list` is introduced to display the suppressed categories of each severity of ASIC/SDK health events.
+
+An example of the output is as below:
+
+```
+admin@sonic:~$ show asic-sdk-health-events suppressed-category-list
```

Comment:
If this command is applicable to multi-asic platforms, please capture the relevant show command examples as well.

Reply:
There is `--namespace` option. By default it's for all ASICs